src/xrpld/app/tx/detail/SetTrust.cpp

```diff
@@ -26,6 +26,42 @@
 #include <xrpl/protocol/Quality.h>
 #include <xrpl/protocol/st.h>
 
+namespace {
+
+uint32_t
```

Comment:
can be static method

Reply:
It's behind anonymous namespace so basically it's doing the same thing already